SWRevealViewController/SWRevealViewController.m

```diff
@@ -896,6 +899,10 @@ - (UIPanGestureRecognizer*)panGestureRecognizer
     return _panGestureRecognizer;
 }
 
+- (void)removeGestureRecognizers {
+    _panGestureRecognizer = nil;
+    _tapGestureRecognizer = nil;
+}
```

Comment:
At this point recognizers are still attached to the views and will trigger their actions if they recognize anything. What is the purpose of this method actually?